worker/src/unstract/worker/worker.py

```diff
@@ -26,19 +26,29 @@ def __init__(self, image_name: str, image_tag: str) -> None:
         #   the Docker daemon in the host environment
         self.client: DockerClient = docker.from_env()  # type: ignore[attr-defined]  # noqa: E501
 
-        private_registry_password = os.getenv(Env.PRIVATE_REGISTRY_PASSWORD)
+        private_registry_credential_path = os.getenv(
+            Env.PRIVATE_REGISTRY_CREDENTIAL_PATH
+        )
         private_registry_username = os.getenv(Env.PRIVATE_REGISTRY_USERNAME)
         private_registry_url = os.getenv(Env.PRIVATE_REGISTRY_URL)
         if (
-            private_registry_password
+            private_registry_credential_path
             and private_registry_username
             and private_registry_url
         ):
-            self.client.login(
-                username=private_registry_username,
-                password=private_registry_password,
-                registry=private_registry_url,
-            )
+            try:
+                open(private_registry_credential_path)
+                self.client.login(
+                    username=private_registry_username,
+                    password=open(private_registry_credential_path).read(),
+                    registry=private_registry_url,
+                )
```

Comment:
Don't we have to handle auth exception variants also for self.client.login()?

Reply:
Addressed.